tests/manual-test-cases/Group13-vMotion/13-2-vMotion-Container.robot

```diff
@@ -17,7 +17,12 @@ Documentation  Test 13-2 - vMotion Container
 Resource  ../../resources/Util.robot
 Suite Setup  Wait Until Keyword Succeeds  10x  10m  Create a VSAN Cluster  vic-vmotion-13-2
 Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
-Test Teardown  Cleanup VIC Appliance On Test Server
+Test Teardown  Cleanup VIC Appliance And Gather VC Logs
+
+*** Keywords ***
+Cleanup VIC Appliance And Gather VC Logs
```

Comment:
Can we avoid duplicating this keyword?

Since this probably isn't useful enough to warrant a "global" keyword in `Vsphere-Util.robot`, maybe we could re-use the pattern established by `tests/resources/Group23-VIC-Machine-Service-Util.robot` and add a `tests/resources/Group13-vMotion-Util.robot` to define this keyword so that it can be used by both of this suites within the group?

In general, I think reducing duplication while minimizing keyword scope will make our robot tests more readable.

Reply:
I guess it's probably better to move this keyword to `VCH-Util.robot`? Collecting VC logs is expensive but we probably would want that in other test suites as well in the future
From a responsibility assignment perspective, this keyword is more like a general test teardown that doesn't limit to Group13

Reply:
`VCH-Util.robot` seems fine!